FSharp.AutoComplete/Program.fs

```diff
@@ -366,7 +370,7 @@ module internal Main =
      new RangeConverter() :> JsonConverter
     |]
 
-  let prAsJson o = printAgent.WriteLine (JsonConvert.SerializeObject(o, jsonConverters))
+  let prAsJson o = printAgent.WriteLine (JsonConvert.SerializeObject(o, Formatting.Indented,  jsonConverters))
```

Comment:
This prints indented, instead of trying to indent later in integration tests. Hope this is ok?